src/plugins/console/public/application/containers/remote/console_header.tsx

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import { EuiFlexGroup, EuiFlexItem, EuiIcon, EuiTitle } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+export const ConsoleHeader = ({
+  isConsoleOpen,
+  rightSideItem,
+  onClick,
+}: {
+  isConsoleOpen: boolean;
+  rightSideItem?: React.ReactNode;
+  onClick?: () => void;
+}) => (
+  <EuiFlexGroup justifyContent="spaceBetween" onClick={onClick}>
```

Comment:
> Any UI touched in this PR is usable by keyboard only (learn more about [keyboard accessibility](https://webaim.org/techniques/keyboard/))

Hey @TattdCodeMonkey. Just curious if you're planning to circle back to this? I'm not certain this is keyboard navigable in its current state.

You may need to do something like add a `tabindex="0"` or use a `button`.

In its current state it's also not apparent that it's clickable, because the cursor doesn't change.

Reply:
This is on my `TODO` list of investigate, which is why I left that `- [ ]` Let me see about rendering this with a button. I had a Panel at one point and that was causing some other side effects that were not ideal.

Reply:
Ah, OK. Sorry, I didn't mean to comment too early. I just saw the PR was out of Draft so thought it might be ready for comment!

Reply:
Thanks for the comment! 

I ended up going back to wrapping this in a Panel for now. I may still need to find another solution as I'm seeing a small color inconsistency on the outsides of the panel. But i'll tackle that after I fix some other pressing issues.